docs/user-guide/command-line-interface.md

```diff
@@ -39,6 +39,7 @@ Options:
   --no-color                  Disable color in piped output
   -o, --output-file path::String  Specify file to write report to
   --quiet                     Report errors only - default: false
+  --max-warnings Number       Maximum warning count before eslint exits with an error (or -1 for no limit)
```

Comment:
Mmm... options seems to be all over the place, sometimes it's name::type, sometimes it's just type. We should probably change it to follow the same style.